frontend/src/router.js

```diff
@@ -13,12 +13,22 @@ export default new Router({
       component: Home
     },
     {
-      path: '/about',
-      name: 'about',
+      path: '/course',
```

Comment:
what will be displayed on `/course`?

Reply:
Wasn't there some vague thing that would be like a newsfeed?

Reply:
Do you know the answers to this question in X or Y course rather than just specific.

Reply:
Hmmm.... even so it's pretty out of scope for now. And I guess the route would be `/courses` and have its own view